### PR TITLE
What's New for 0.21.0

### PR DESCRIPTION
**Pull request by pmhayward**
Update to latestRelease.md

Signed-off-by: Peter Hayward <pmhayward@uk.ibm.com>

### CODE REVIEW

File: news-page-content/latestRelease.md [hunk 1]
```diff
@@ -22,20 +22,32 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
...
+
+- `-XX:[+|-]PrintFlagsFinal` outputs the values of a subset of configuration parameters in a format compatible with that produced by HotSpot. This is an initial implementation, and over time, we expect more options to be added to the output.
+
+And for your convenience, the Application Programming Interface (API) documentation that applies to OpenJ9 can now be found in the user documentation for both JDK 8 and JDK 11. The documentation includes links to Oracle API documentation for information that is not specific to OpenJ9.
+
+On macOS&reg; systems: You should note that in this release, the version information for shared libraries on macOS has been updated from 0.0.0 to 1.0.0. If an application has linked against a shared library from a previous OpenJ9 release, it needs to be re-linked against the new release. Failure to re-link causes an error `Incompatible library version`, `requires version 0.0.0`.
+
+For all the details of changes and improvments in 0.21.0, read the [Version 0.21.0 "What's New" page](https://www.eclipse.org/openj9/docs/version0.21/) and see also the [OpenJ9 Release notes](https://github.com/eclipse/openj9/blob/master/doc/release-notes/0.21/0.21.md).
```
Comment (SueChaplain) [inline news-page-content/latestRelease.md:53]: improvments/improvements

File: news-page-content/latestRelease.md [hunk 2]
```diff
@@ -22,20 +22,32 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
...
 
-OpenJ9 Version 0.20.0 also contains a few improvements to existing command line options, some of which change behavior, so make sure you read the [Version 0.20.0 release notes](https://www.eclipse.org/openj9/docs/version0.20/). If you are obtaining OpenJDK 8 or 11 from the AdoptOpenJDK community, you should also read the [Version 0.19.0 release notes](https://www.eclipse.org/openj9/docs/version0.19/) to learn about all the other features and changes in the VM since their last OpenJDK 8 and 11 binaries were made available.
\ No newline at end of file
+- Several performance features were added to the AArch64 JIT compiler implementation that led to a throughput improvement on multiple applications of at least 20%. The most notable improvements were seen in global register allocation, recompilation (without profiling), CUDA support, concurrent scavenge GC policy, and the inlined code sequence for object allocations.
+
+In this release, we've introduced two new options. See the user documentation for more information about both.
```
Comment (SueChaplain) [inline news-page-content/latestRelease.md:43]: I think this reads better without the "See the user documentation for more information about both." We tell users to go to the user docs at the end, so no loss.

File: news-page-content/latestRelease.md (hunk 1 above)
Comment (pmhayward) [reply news-page-content/latestRelease.md]: Fixed!

File: news-page-content/latestRelease.md (hunk 2 above)
Comment (pmhayward) [reply news-page-content/latestRelease.md]: Removed!

**Pull request merged**